### ISSUE TITLE
OKE worker nodes boot_volume_size not expanded

### ISSUE THREAD

**Issue by iris-garcia** [oracle-terraform-modules/terraform-oci-oke#414]
### Terraform Version and Provider Version
Terraform: `v1.0.7`
Provider: `4.41.0`
terraform-oci-oke: `3.3.0`

### Expected Behavior
When creating a Kubernetes cluster specifying a node_pool like this:
```
  node_pools = {
    np1 = {
      boot_volume_size = 150
      node_pool_size   = 3
      ocpus            = 2
      memory           = 8
      shape            = "VM.Standard.E3.Flex"
      label = {
        nodepool = "np1"
      }
    }
  }
```
The node's space capacity should match the boot_volume_size.
I would expect somewhere in the code to run `growpart`.

### Actual Behavior
The root partition is always 50GB and the rest of the space is not used.
By running the command `growpart /dev/sda 3` and rebooting the node the partition uses its whole capacity.

### Steps to Reproduce

Just create a cluster with the boot_volume_size greater than 50GB and check the Size of each partition

**Comment by hyder** [comment]
Hi, is this issue happening when you are creating the nodepool for the first time?

**Comment by iris-garcia** [comment]
Hi @hyder, I created a cluster with one nodepool and it didn't get expanded, then I added a second nodepool which did not get expanded either.

**Comment by karthicgit** [comment]
We are not extending the volume as part of the module it's just setting the custom boot volume size.
If you specify a custom boot volume size, you need to extend the volume to take advantage of the larger size.(https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/bootvolumes.htm#Custom)

**Comment by iris-garcia** [comment]
Hi @karthicgit, I realized about that, in my initial message I pointed the steps I took to extend the partition.

But this is far from ideal, we need to run terraform to create the cluster then use a workaround to ssh into every single node, extend the partition and rebot the node.

I thought you had planned in the roadmap an automated way of handling this, like cloudinit script runing the command or even another terraform module.
